src/types/entities/schelling.ts

```diff
@@ -138,13 +210,24 @@ export class SchellingGame {
 	// --- claim
 	public claim(
 		winner: Reveal,
+		owner: string,
 		amount: BigNumber,
 		block: BlockDetails,
 		freezes: StakeFreeze[] = [],
 		slashes: StakeSlash[] = []
 	) {
 		const roundNo = SchellingGame.roundFromBlockNo(block.blockNo)
 
+		const round = this.getOrCreateRound(roundNo, block)
+
+		// create the player if it doesn't exist
+		if (!this.players.has(winner.overlay)) {
+			this.players.set(
+				winner.overlay,
+				new Player(winner.overlay, owner, block, this.size)
+			)
+		}
+
```

Comment:
This should be impossible if we enforce the invariant of starting syncing from the initial block of a round.

Reply:
"should" is one of my hot words.   If it can't happen, it probably will sometime, so the code should handle it if it is conveniently possible.  And I don't want the code to "trust" that it sees all of the required transactions in order, but if it sees a new player that showed up in the win, then that's still a Player to be captured and displayed.